include/odp/api/spec/pool.h

```diff
@@ -294,6 +294,19 @@ odp_pool_t odp_pool_lookup(const char *name);
 typedef struct odp_pool_info_t {
 	const char *name;          /**< pool name */
 	odp_pool_param_t params;   /**< pool parameters */
+	/** Minimum data address.
+	This is the minimum address that application accessible data of any
+	object (event) allocated from the pool may locate. When there's no
+	application accessible data (e.g. ODP_POOL_TIMEOUT pools), the
+	value maybe zero. */
+	uintptr_t min_data_addr;
+
+	/** Maximum data address.
+	This is the maximum address that application accessible data of any
+	object (event) allocated from the pool may locate. When there's no
+	application accessible data (e.g. ODP_POOL_TIMEOUT pools), the
+	value maybe zero. */
+	uintptr_t max_data_addr;
```

Comment:
While adding commit log text, modify the style in these comments to begin each row with a *.

'/**
  *
  *
  */'

Reply:
Sure.